dbms/src/Storages/KVStore/TMTContext.h

```diff
@@ -173,6 +175,8 @@ class TMTContext : private boost::noncopyable
     std::atomic_uint64_t batch_read_index_timeout_ms;
     std::atomic_uint64_t wait_index_timeout_ms;
     std::atomic_uint64_t read_index_worker_tick_ms;
```

Comment:
Move all those read-index related fields into KVStore, and then introduce a `KVStore::reloadConfig`? Thus make the TMTContext clearer, and the KVStore module more cohesive.